src/data/complex/exponential.lean

```diff
@@ -1470,18 +1470,15 @@ calc cos 1 ≤ abs' (1 : ℝ) ^ 4 * (5 / 96) + (1 - 1 ^ 2 / 2) :
   sub_le_iff_le_add.1 (abs_sub_le_iff.1 (cos_bound (by simp))).1
 ... ≤ 2 / 3 : by norm_num
 
-lemma cos_one_pos : 0 < cos 1 := cos_pos_of_le_one (by simp)
+lemma cos_one_pos : 0 < cos 1 := cos_pos_of_le_one (le_of_eq abs_one)
```

Comment:
```suggestion
lemma cos_one_pos : 0 < cos 1 := cos_pos_of_le_one abs_one.le
```

Reply:
For some reason, the abbreviation does not work here...